MediaBrowser.Controller/MediaEncoding/IMediaEncoder.cs

```diff
@@ -187,5 +195,27 @@ public interface IMediaEncoder : ITranscoderSupport
         /// <param name="path">The path.</param>
         /// <param name="pathType">The type of path.</param>
         void UpdateEncoderPath(string path, string pathType);
+
+        /// <summary>
+        /// Gets the primary playlist of .vob files.
+        /// </summary>
+        /// <param name="path">The to the .vob files.</param>
+        /// <param name="titleNumber">The title number to start with.</param>
+        /// <returns>A playlist.</returns>
+        IEnumerable<string> GetPrimaryPlaylistVobFiles(string path, uint? titleNumber);
```

Comment:
```suggestion
        IReadOnlyList<string> GetPrimaryPlaylistVobFiles(string path, uint? titleNumber);
```

I think we prefer to avoid `IEnumerable` unless there is a real benefit